app/Models/Matter.php

```diff
@@ -339,6 +339,7 @@ public static function filter($sortkey = 'id', $sortdir = 'desc', $multi_filter
     {
         $query = Matter::select(
             'matter.uid AS Ref',
+            'matter.alt_ref AS Alt_Ref',
```

Comment:
I have an issue with this being in second place, since it will essentially be empty for most users. Put it at the end.